addons/missionModules/functions/fnc_zeus_banziCharge.sqf

```diff
@@ -0,0 +1,15 @@
+#include "script_component.hpp"
+
+params ["_logic"];
+
+if (!local _logic) exitWith {};
+private _unit = _logic getVariable ["bis_fnc_curatorAttachObject_object",objnull];
+TRACE_2("",_logic,_unit);
+
+if ((!isNull _unit) || {alive _unit} || {_unit isKindOf "CaManBase"}) then {
+    [QGVAR(banzi), [group _unit], [leader _unit]] call CBA_fnc_targetEvent;
```

Comment:
Just curious, is there any advantage to using an event system here, instead of remoteexec-ing? If we don't plan on having hooks in other addons, it seems like we're using a lot of  piping for a simple call